man/repart.d.xml

```diff
@@ -580,6 +580,23 @@
         <para>This option has no effect if the partition already exists.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Verity=</varname></term>
+
+        <listitem><para>Takes one of <literal>data</literal> or <literal>hash</literal>. Defaults to
+        <literal>data</literal>. If set to <literal>data</literal>, the partition will be populated with
```

Comment:
I'd like to also add `"no"` or `"off"`and defaults to that. For safety to make not partitions mistakenly detected as a data partition for a hash partition.